src/trajectory_tracking.h

```diff
@@ -0,0 +1,61 @@
+
+#ifndef __TRACY__
+#define __TRACY__
+
+#include <stdint.h>
+
+#define DEFAULT_PARAM_DAMPING_COEFF (0.7f)
+#define DEFAULT_PARAM_G (60.0f)
+
+/**
+ * output of the linear controller
+ *
+ */
+typedef struct {
+    float tangential_velocity;
+    float angular_velocity;
+} velocity_command_t;
```

Comment:
I suggest to define a struct instead of a typedef, since we don't provide getter functions.